clang/include/clang/Driver/Driver.h

```diff
@@ -450,6 +454,11 @@ class Driver {
     return ClangExecutable.c_str();
   }
 
+  StringRef getPreferredLinker() const { return PreferredLinker; }
+  void setPreferredLinker(std::string Value) {
```

Comment:
If #149786 will use `setPreferredLinker`, this `setPreferredLinker` (currently unused) is ok.